test/integration/bastion/bastion_test.go

```diff
@@ -126,11 +125,7 @@ var _ = BeforeSuite(func() {
 	alicloud.InternalChartsPath = filepath.Join(repoRoot, alicloud.InternalChartsPath)
 
 	// enable manager logs
-	logf.SetLogger(zap.New(zap.UseDevMode(true), zap.WriteTo(GinkgoWriter)))
-
-	log := logrus.New()
-	log.SetOutput(GinkgoWriter)
-	logger = logrus.NewEntry(log)
+	logger = log.Log.WithName("test")
```

Comment:
Please adapt similar to https://github.com/gardener/gardener-extension-provider-aws/pull/585/commits/b01bfb96b3c8fb3e0267a1715cd4171a3016ac32

Reply:
a42163db done

Reply:
You can cleanup the `logger` from actuator structs, wherever possible. ref: https://github.com/gardener/gardener/pull/6332/commits/1a3570f086eacf77daa2c297d9296bad86dfef0a

Reply:
5ce0129e  cleanup `logger` from bastion actuator structs.  let me know if correct, besides, should I create one separate ticket for cleanup `logger` for other objects under the controller folder?

Reply:
4259a5bae9679cbb43aed45ac961dceeaefde2fc
495d17b692f2c508b0c09b24fcd628a1eb41f544